open_spiel/python/examples/deep_cfr_pytorch.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python Deep CFR example."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl import app
+from absl import flags
+from absl import logging
+import six
+
+import tensorflow.compat.v1 as tf
+
+from open_spiel.python import policy
+from open_spiel.python.pytorch import deep_cfr
+from open_spiel.python.algorithms import exploitability
+import pyspiel
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer("num_iterations", 400, "Number of iterations")
+flags.DEFINE_integer("num_traversals", 40, "Number of traversals/games")
+flags.DEFINE_string("game_name", "kuhn_poker", "Name of the game")
+
+
+def main(unused_argv):
+  logging.info("Loading %s", FLAGS.game_name)
+  game = pyspiel.load_game(FLAGS.game_name)
+
+  deep_cfr_solver = deep_cfr.DeepCFRSolver(
+      game,
+      policy_network_layers=(32, 32),
+      advantage_network_layers=(16, 16),
+      num_iterations=FLAGS.num_iterations,
+      num_traversals=FLAGS.num_traversals,
+      learning_rate=1e-3,
+      batch_size_advantage=None,
+      batch_size_strategy=None,
+      memory_capacity=1e7)
+  
+  _, advantage_losses, policy_loss = deep_cfr_solver.solve()
+  
+  for player, losses in six.iteritems(advantage_losses):
+    logging.info("Advantage for player %d: %s", player,
+                  losses[:2] + ["..."] + losses[-2:])
+    logging.info("Advantage Buffer Size for player %s: '%s'", player,
+                  len(deep_cfr_solver.advantage_buffers[player]))
+  logging.info("Strategy Buffer Size: '%s'",
+                len(deep_cfr_solver.strategy_buffer))
+  logging.info("Final policy loss: '%s'", policy_loss)
+  conv = exploitability.nash_conv(
```

Comment:
Can you use the C++ exploitability instead (pyspiel.nash_conv)? It's much faster.